### PR TITLE
Update index.d.ts 

### PR DESCRIPTION
**Pull request by nazarhussain**
## Description

Add the CHANGELOG for the long awaited community PR https://github.com/ChainSafe/web3.js/pull/4955

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [x] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [x] I ran `npm run build` with success.
- [x] I have tested the built `dist/web3.min.js` in a browser.
- [x] I have tested my code on the live network.
- [x] I have checked the Deploy Preview and it looks correct.
- [x] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -580,6 +580,7 @@ Released with 1.0.0-beta.37 code base.
 -  Add optional hex formatting parameter for getTransactionrReceipt (#5153)
 -  Fix transactionRoot -> transactionsRoot in BlockHeader (#5083)
 -  Fix Promise in Accounts.signTransaction() throwing errors that cannot be caught (#4724)
+-  Fixed types for getPastEvents
```
Comment (jdevcs) [inline CHANGELOG.md]: 1.7.5 is already frozen under RC so make it under 1.7.6, and add label future release

**Review by jdevcs — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -584,3 +584,13 @@ Released with 1.0.0-beta.37 code base.
...
 -  Updated `got` lib version and fixed other libs using npm audit fix (#5178) (#5254)
 
+## [Unreleased]
+
+## [1.7.6]
+
+### Changed
+
+### Fixed
+-  Fixed types for getPastEvents
```
Comment (jdevcs) [inline CHANGELOG.md]:
```suggestion
-  Fixed types for getPastEvents (#4955) (#5260)
```

**Review by jdevcs — CHANGES_REQUESTED** [review]

**Review by jdevcs — APPROVED** [review]

**Review by avkos — APPROVED** [review]

File: packages/web3-eth-contract/types/index.d.ts [hunk 3]
```diff
@@ -68,12 +68,12 @@ export class Contract {
     getPastEvents(
         event: string,
         options: PastEventOptions,
-        callback: (error: Error, event: EventData) => void
+        callback: (error: Error, event: EventData[]) => void
```
Comment (Muhammad-Altabba) [inline packages/web3-eth-contract/types/index.d.ts]:
```suggestion
        callback: (error: Error, events: EventData[]) => void
```

File: packages/web3-eth-contract/types/index.d.ts [hunk 4]
```diff
@@ -68,12 +68,12 @@ export class Contract {
...
     ): Promise<EventData[]>;
     getPastEvents(event: string, options: PastEventOptions): Promise<EventData[]>;
     getPastEvents(
         event: string,
-        callback: (error: Error, event: EventData) => void
+        callback: (error: Error, event: EventData[]) => void
```
Comment (Muhammad-Altabba) [inline packages/web3-eth-contract/types/index.d.ts]:
```suggestion
        callback: (error: Error, events: EventData[]) => void
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -584,3 +584,13 @@ Released with 1.0.0-beta.37 code base.
 ### Security
 -  Updated `got` lib version and fixed other libs using npm audit fix (#5178) (#5254)
 
+## [Unreleased]
```
Comment (Muhammad-Altabba) [inline CHANGELOG.md]: As the previous version (`1.7.5`) was not released yet, is it possible to add those changes to `1.7.5-rc.0`? Or would that violate the process?

**Review by Muhammad-Altabba — COMMENTED** [review]
Looks good. However, I have a small question and tiny suggestions.
Thanks,

File: CHANGELOG.md (hunk 5 above)
Comment (nazarhussain) [reply CHANGELOG.md]: It was code freeze from our perspective, so we can't make any changes to it. The current workflow is to include the PR's in future release during the code freeze duration.

**Review by Muhammad-Altabba — APPROVED** [review]
:+1:

**Comment by wbt** [comment]
Thanks much! l look forward to seeing this in a release soon.

**Pull request merged**